### PR TITLE
spring boot task

### PR DESCRIPTION
**Pull request by abhinay710**

### CODE REVIEW

File: Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/Employee.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+public class Employee {
+    @Id
+    @Column(name="emp_id")
+    private Long id;
+    @Column(name="emp_name")
+    private String employeeName;
+    @Column(name="emp_description")
+    private String designation;
+    @Column(name="emp_dob")
+    private String dob;
```
Comment (manojkustommade) [inline Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/Employee.java]: change variable names like dateOfBirth

File: Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    @Autowired
+    private EmployeeService employeeService;
+
+    @RequestMapping("/employee")
+    public List<Employee>  getEmployeeService(){
+        return employeeService.getEmployeeRepository();
+    }
+
+    @PostMapping("/employee")
+    public void setEmployeeService(@RequestBody Employee employees ){
```
Comment (manojkustommade) [inline Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeController.java]: rename to addEmployee

File: Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        employeeService.setEmployeeRepository(employees);
+    }
+
+    @PutMapping("/employee/{id}")
+    public void updateEmployeeService(@RequestBody Employee employees ,@PathVariable Long id){
+        employeeService.setEmployeeRepository(employees);
```
Comment (manojkustommade) [inline Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeController.java]: rename to updateEmployee

File: Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeService.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+import java.util.List;
+
+@Service
+public class EmployeeService {
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+    public void setEmployeeRepository(Employee employees){
```
Comment (manojkustommade) [inline Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/EmployeeService.java]: create EmployeeService Interface and implement meothds

File: Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/Employee.java (hunk 1 above)
Comment (abhinay710) [reply Spring Project/employee/src/main/java/com/mydezinertraining/springboot/employee/Employee.java]: ok

**Pull request closed without merging**